contrib/forms/assessment_intake/view.php

```diff
@@ -376,4 +394,4 @@
 </form>
 <?php
 formFooter();
-?>
+?> 
```

Comment:
Yeah it's a bit odd that that all happened. I think the next step is tweaking phpcs.xml to add the rest of the PSR2 rules and some custom ones of our own, since it seems to be applying very few. I have no idea why it threw those spaces in there (or why that's not an error...).